.travis.yml

```diff
@@ -20,7 +20,7 @@ compiler: gcc
 env:
     - BUILD_TYPE=debug
     - BUILD_TYPE=release
-sudo: required
+dist: xenial
```

Comment:
Why did you do this? Was missing before or it's a new requirement?

Reply:
It was required to get travis tests to pass. I don't really know why I had to force travis to use xenial, as https://blog.travis-ci.com/2019-04-15-xenial-default-build-environment says xenial is the default if unspecified. The commit msg for https://github.com/falcosecurity/falco/pull/826/commits/4c820a678424d0b00b74e99f4833b32c971ce1f2 has a bit more detail.